manim/mobject/probability.py

```diff
@@ -251,7 +252,7 @@ def __init__(
         label_y_axis: bool = True,
         y_axis_label_height: float = 0.25,
         max_value: float = 1,
-        bar_colors: List[Color] = [BLUE, YELLOW],
+        bar_colors: List["Color"] = [BLUE, YELLOW],
```

Comment:
Are you sure the quotes are correct? It seems to be unquoted in other typehints in the rest of the library.

Reply:
Yeah it works only when you have quotes and we need to change all the other instances in the code and the documentation for the typings also.

Reply:
Sorry but I'm not quite convinced that it is correct. Could you explain why we're using a string instead of the actual typing import?

Reply:
I think we are supposed to use `if typing.TYPE_CHECKING:  ` as @kolibril13 stated above for importing Color module as it needed only for the typings and for this to work we need to put it as a string. If we don’t it errors out as shown [here](https://github.com/ManimCommunity/manim/pull/1642/checks?check_run_id=2779961937)